assets/models/system/automl-image-classification/description.md

```diff
@@ -10,28 +10,27 @@ With this functionality, you can:
 
 See [How to train image models](https://learn.microsoft.com/azure/machine-learning/how-to-auto-train-image-models?view=azureml-api-2&tabs=cli) for more information.
 
-### Documentation
+# Training Details
 
-#### Prepare Data
```

Comment:
I think, this is most important section. Let's have it in heading training data.

Reply:
adding Training Data section instead of prepare data to align with conventions